src/scripts/content/zendesk.js

```diff
@@ -43,7 +43,7 @@ togglbutton.render('[data-test-id="customer-context-tab-navigation"]', { observe
   const titleFunc = function () {
     let description;
 
-    const titleElem = document.querySelector('[aria-label="Subject"]');
+    const titleElem = document.querySelector('[data-test-id="ticket-pane-subject"]');
```

Comment:
`data-test-id` 😂 

Question is when do Zendesk team realise and strip these from production. 🤔

Reply:
Wondering if isn't intentional, but this is the only unique selector that I found. I hope they keep it 🙈

Reply:
So far so good! Maybe they will never switch it off 🙊